Formula/pipectl.rb

```diff
@@ -0,0 +1,51 @@
+# typed: false
+# frozen_string_literal: true
+
+class Pipectl < Formula
+    desc "pipectl: A Command Line Tool for PipeCD."
+    homepage "https://github.com/pipe-cd/pipecd"
+    version "0.47.0"
```

Comment:
We need to update this version with each new release. Although this process can be automated, it is beyond the scope of this PR.

Reply:
> Although this process can be automated, it is beyond the scope of this PR.
Agree with your idea, I think we can set up a GitHub action workflow to sync version of pipectl from main repo to homebrew-pipectl repo 👍 
(Just like we do to sync the examples repo from main repo like this: https://github.com/pipe-cd/examples/blob/master/.github/workflows/sync.yaml )